### ISSUE TITLE
The isolated form of glyph tcheh-ar/tcheheh-ar is missing it's three dots/four dots.

### ISSUE THREAD

**Issue by graphicore** [Tarobish/Katibeh#94]
This is uni0686. It should have 3 dots.
uni0687 looks the same and should have 4 dots
Same applies to Mirza.

![selection_058](https://cloud.githubusercontent.com/assets/393132/12920589/835a573e-cf49-11e5-9c71-b78de6caf24a.png)

![selection_059](https://cloud.githubusercontent.com/assets/393132/12920702/ed8253b4-cf49-11e5-85ca-4e5749ecc177.png)

**Comment by graphicore** [comment]
There are more broken glyphs of the family that has HAH as bases:

JEEM has one dot in the center, but that is also misiing from `uni062C`it's final form uniFE9E is also missing the dot.

Seems like this issue is affecting many glyphs!

**Comment by graphicore** [comment]
I'm a bit puzzled. It must have been correct once, because I was able to get the decompositions of these glyphs. But now all of them are broken. Take also a look at the `fina` forms.

The marks are there for the init and medi forms at it seems.

This is related to #82 by the way. I'm working on that and make these findings here.
Some of the decompositions need to be made differently I think, I'm working on a viable solutions.

**Comment by graphicore** [comment]
Do you need a list of these glyphs, where the center mark is missing?

I can make one for these that we use in decomposition at least.

**Comment by graphicore** [comment]
Aha! This is a false alert!

CC @Schriftgestalt

I was looking in the ufos exported by Glyphs.
Glyphs doesn't translate all of the component names correctly into production names when exporting the ufo:

This is https://github.com/Tarobish/Katibeh/blob/master/Generated/Arabic/Katibeh-Regular.ufo/glyphs/uni0686.glif:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<glyph name="uni0686" format="1">
  <advance width="575"/>
  <unicode hex="0686"/>
  <outline>
    <component base="uni062D"/>
    <component base="threedotsdowncenter-ar" xOffset="330" yOffset="-282"/>
  </outline>
</glyph>
```

the second component name should in fact be `threedotsdowncenterar` without the hyphen.

**Comment by Tarobish** [comment]
I don't want the list, ill check them all thru "Glyph" view from indesign

**Comment by graphicore** [comment]
@Tarobish Here's probably nothing more to do for you. But yeah, maybe double-check just in case.

**Comment by schriftgestalt** [comment]
> the second component name should in fact be threedotsdowncenterar without the hyphen.

This should be fixed in Glyphs 2.3.
